zerver/tests/test_events.py

```diff
@@ -258,6 +260,8 @@ def match_states(self, state1, state2):
         def normalize(state):
             # type: (Dict[str, Any]) -> None
             state['realm_users'] = {u['email']: u for u in state['realm_users']}
+            for u in state['subscriptions']:
+                u['subscribers'].sort()
```

Comment:
@neerajwahi Even I was confused about this and was blocked. Since order of subscribers doesn't affect any other functionality, I added it here.